web/src/test/java/com/replica/StreetEdgeExporterTest.java

```diff
@@ -0,0 +1,95 @@
+import com.graphhopper.http.GraphHopperApplication;
+import com.graphhopper.http.GraphHopperBundle;
+import com.graphhopper.http.GraphHopperServerConfiguration;
+import com.graphhopper.http.cli.ExportCommand;
+import com.graphhopper.http.cli.ImportCommand;
+import com.graphhopper.replica.StreetEdgeExporter;
+import com.graphhopper.util.Helper;
+import io.dropwizard.cli.Cli;
+import io.dropwizard.setup.Bootstrap;
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
+import io.dropwizard.testing.junit5.DropwizardExtensionsSupport;
+import io.dropwizard.util.JarLocation;
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+
+import org.apache.commons.csv.CSVRecord;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(DropwizardExtensionsSupport.class)
+public class StreetEdgeExporterTest {
+
+    private static final String TARGET_DIR = "./target/gtfs-app-gh/";
+    private static final String TRANSIT_DATA_DIR = "transit_data/";
+    public static final DropwizardAppExtension<GraphHopperServerConfiguration> app = new DropwizardAppExtension<>(GraphHopperApplication.class, createConfig());
+    private static GraphHopperServerConfiguration createConfig() {
+        GraphHopperServerConfiguration config = new GraphHopperServerConfiguration();
+        config.getGraphHopperConfiguration().
+                putObject("graph.flag_encoders", "car,foot").
+                putObject("datareader.file", "test-data/beatty.osm").
+                putObject("gtfs.file", "test-data/sample-feed.zip").
+                putObject("graph.location", TARGET_DIR).
+                setProfiles(Collections.singletonList(new Profile("car").setVehicle("car").setWeighting("fastest")));
+        return config;
+    }
+
+    @BeforeAll
+    public static void setUp() {
+        // Fresh target directory
+        Helper.removeDir(new File(TARGET_DIR));
+        // Create new empty directory for GTFS/OSM resources
+        File transitDataDir = new File(TRANSIT_DATA_DIR);
+        if (transitDataDir.exists()) {
+            throw new IllegalStateException(TRANSIT_DATA_DIR + " directory should not already exist.");
+        }
+        Preconditions.checkState(transitDataDir.mkdir(), "could not create directory " + TRANSIT_DATA_DIR);
+    }
+
+    @AfterAll
+    public static void cleanUp() {
+        Helper.removeDir(new File(TARGET_DIR));
+        Helper.removeDir(new File(TRANSIT_DATA_DIR));
+    }
+
+    @Test
+    public void testSomething() throws IOException {
```

Comment:
looks good.  but maybe rename the test to testExport() or something more descriptive?

Reply:
Oh, whoops, you caught me, that was just what I put down when I was getting started. Thanks!